feature-libs/asm/core/store/actions/customer.action.ts

```diff
@@ -44,9 +56,41 @@ export class CustomerSearchReset extends StateUtils.LoaderResetAction {
   }
 }
 
+export class CustomerListCustomersSearch extends StateUtils.LoaderLoadAction {
```

Comment:
This action.ts looks good me, just one small thing: Why `CustomerListCustomersSearch`? (there are 2 customer in the name). how about `CustomerListSearch`?